docs/usage/configuration-options.md

```diff
@@ -466,6 +466,18 @@ If configured to `true`, then dependency extraction will be done using the relev
 
 This option applies only to the `gradle` manager.
 
+## defaultRegistryUrls
+
+This configuration option enables overriding of a datasource's default registries.
+Like `registryUrls`, it is only applicable when the datasource's `customRegistrySupport` value is set to `true`.
+
+Default registries are only used when both:
+
+- The manager does not extract any `registryUrls` values, and
```

Comment:
```suggestion
- The manager did not extract any `registryUrls` values, and
```

The next list item uses the passive tense. So we should use `did` here to be consistent, I think.